src/Endpoints/RabbitHoleEndpoint.php

```diff
@@ -126,11 +130,9 @@ public function postWeb(
         if ($chunkOverlap) {
             $payload['chunk_overlap'] = $chunkOverlap;
         }
-
-        $multipartData[] = [
-            'name' => 'metadata',
-            'contents' => json_encode($metadata)
-        ];
+        if ($metadata) {
```

Comment:
it should be like this, since there is no multipart/form-data content in the CheshireCat's endpoint AI-Blackbird:bugfix/rabbithole